wiki_articles/vscode.md

```diff
@@ -0,0 +1,33 @@
+# Getting Started With VS Code for C/C++
+
+## DISCLAIMER FOR BEGINNERS
+Do not use VS Code.
+It's better to use an IDE that works out of the box:
+- [Visual Studio Community](https://visualstudio.microsoft.com/vs/community/) for Windows (free)
+- [CLion](https://www.jetbrains.com/clion/) on any platform (free for students)
+- [XCode](https://developer.apple.com/xcode/) on macOS (free)
+- [repl.it](https://replit.com/site/ide) online IDE (free for public projects)
```

Comment:
This needs to be softened. I'd recommend
```suggestion
## Disclaimer for Beginners
VSCode requires a lot of configuration for C/C++ development. Some IDEs that are much faster and easier to set up include [Visual Studio Community](https://visualstudio.microsoft.com/vs/community/) (free, Windows), [CLion](https://www.jetbrains.com/clion/) (free for students), [XCode](https://developer.apple.com/xcode/) (free, macOS), and [repl.it](https://replit.com/site/ide) (free for public projects, online IDE).
```

Reply:
Firstly, I would want to keep it formatted as a list regardless of how the preceding prose is formulated. Secondly, while the wording is bit harsh, this is very intentional.

The point of this paragraph is that we have at least a few people every week who are struggling with getting VSCode to work for C++, and I also see this during my tutoring sessions. It's a massive waste of everyone's time, and this needs to stop. If we have soft wording like "this requires some setup and you could slightly more setup" then we're not getting the message across not to use VSCode. Beginners need *clear* language, not this wishy washy stuff.

I've received positive feedback on the server for a preview of this article; the consensus on VS Code is not to use it when you're a beginner.

Reply:
I understand what you're aiming for, and I'm worked with plenty of beginners trying to get vscode setup as well. It's ultimately not that hard to setup though. I think good guides for setting it up are lacking online, but that's the gap we should aim to fill. Furthermore, it's very frustrating for beginners to be trying to use something and have people tell them to switch to something else.